tests/server_test.py

```diff
@@ -39,6 +96,15 @@ def test_srw(fc):
         r.assert_http_status(200)
 
 
+def test_srw_light(fc):
+    from pykern.pkunit import pkre
+
+    r = fc.sr_get("srwLight")
+    r.assert_success()
+    pkre("<!DOCTYPE html>", r.data)
```

Comment:
Make a single regex, and can simply check for the existence of SRWLightGateway. The data-ng-app is unnecessary.

Reply:
1a08a18cb5e9c42888b493a7621bd56f9ac7faea